packages/protocol/contracts/governance/Governance.sol

```diff
@@ -1472,14 +1472,20 @@ contract Governance is
 
     uint256 maxUsed = 0;
     for (uint256 index = 0; index < dequeued.length; index = index.add(1)) {
-      Proposals.Proposal storage proposal = proposals[dequeued[index]];
+      uint256 proposalId = dequeued[index];
+      Proposals.Proposal storage proposal = proposals[proposalId];
```

Comment:
Can we create a function for this? I think it should be private so it can return a Porposal in SOlidity 0.5.x

`getProposal(uint256 dequeueIndex)`

Reply:
Lets do this in separate task